.github/workflows/release.yml

```
@@ -0,0 +1,34 @@
+          zip -r dist/schemas.zip schemas
+          zip -r dist/vectors.zip tests/vectors scripts/vectors
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v2
+        with:
+          files: |
+            dist/schemas.zip
+            dist/vectors.zip
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)